hack/generate_krew.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This script generates a Krew-compatible plugin manifest. It should be run after goreleaser.
+
+VERSION=${VERSION:-$(git describe --tags |sed 's/^v//g')}
+
+# Generate the manifest for a single platform.
+function generate_platform {
+    ARCH="${2}"
+    if [ "${2}" == "amd64" ]; then
+        ARCH=x86_64
+    elif [ "${2}" == "386" ]; then
+        ARCH=i386
+    fi
+
+    cat <<EOF
+  - selector:
+      matchLabels:
+        os: "${1}"
+        arch: "${2}"
+    uri: https://github.com/kudobuilder/kudo/releases/download/v${VERSION}/kudo_${VERSION}_${1}_${ARCH}.tar.gz
+    sha256: "$(curl -L https://github.com/kudobuilder/kudo/releases/download/v${VERSION}/kudo_${VERSION}_${1}_${ARCH}.tar.gz |sha256sum - |awk '{print $1}')"
+    bin: "./kubectl-kudo"
+    files:
+    - from: "*"
+      to: "."
+EOF
+}
+
+rm -f kudo.yaml
+
+cat <<EOF >> kudo.yaml
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kudo
+spec:
+  version: "${VERSION}"
+
+  shortDescription: Declaratively build, install, and run operators using KUDO.
```

Comment:
@mattj-io / @yankcrime / @gerred / @runyontr is there someone that would be willing to help me edit the descriptions here?